### PR TITLE
Assign crafter dropdown

### PR DESCRIPTION
**Pull request by m-pitera**
Here we formatted a specific dropdown in the unassigned applicants page. We deleted a js file and there were some css changes were made.

### CODE REVIEW

File: app/assets/stylesheets/base.css.scss [hunk 1]
```diff
@@ -931,7 +934,7 @@ form, .form-column {
 }
 
 label {
-  display: block;
+  display: block;
```
Comment (ssunday) [inline app/assets/stylesheets/base.css.scss]: Extra spacing

Comment (ssunday) [reply app/assets/stylesheets/base.css.scss]: the git highlighting shows it

File: app/assets/javascripts/reporting.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ $(function() {
     var data = {
       labels: window.reportingData["labels"],
       datasets: [{
-        label: "Total Craftsmen",
+        label: "Total Crafters",
```
Comment (ssunday) [inline app/assets/javascripts/reporting.js]: 😬

File: app/controllers/applicants_controller.rb [hunk 3]
```diff
@@ -137,10 +137,14 @@ def unassigned
   end
 
   def assign_craftsman
-    applicant = repo.applicant.find_by_id(params[:id])
-    steward = repo.craftsman.find_by_email(ENV['STEWARD'])
-    ApplicantDispatch::Dispatcher.new(applicant, steward).assign_applicant
-    redirect_to(unassigned_applicants_path, notice: "Assigned #{applicant.name} to #{applicant.assigned_craftsman}")
+    if params[:applicant_to_assign]["chosen_crafter"] == "auto"
```
Comment (ssunday) [inline app/controllers/applicants_controller.rb]: Could auto be a constant or this check be pulled out into a method?

Comment (ssunday) [reply app/controllers/applicants_controller.rb]: like `automatically_assign_crafter?`

File: app/views/applicants/unassigned.html.erb [hunk 4]
```diff
@@ -11,11 +11,11 @@
               <p>Applied <%= pluralize(@applicant_presenter.days_since_last_applied(app.applied_on), "day", "days") %> ago</p>
             </div>
             <span class="applicant-actions">
-              <%= link_to "Auto-Assign", {:controller => 'applicants', :action => 'assign_craftsman', :id => app.id }, {:class => "button primary"} %>
               <%= form_for :applicant_to_assign, url: specify_craftsman_path, method: :post do |f| %>
-                <%= f.select(:chosen_crafter, options_for_select(@craftsmen.map { |crafter| [crafter['name'], crafter['name']] })) %>
-                <%= f.hidden_field(:id, :value => app.id) %>
-                <%= f.submit "Assign to Crafter", class: "button primary" %>
+                <label class="dropdown-label">Assign to:</label>
+                <%= f.select(:chosen_crafter, options_for_select(@craftsmen.map { |crafter| [crafter['name'], crafter['name']] }.unshift(["Available Crafter","auto"])), {}, class: "dropdown-height dropdown") %>
```
Comment (ssunday) [inline app/views/applicants/unassigned.html.erb]: I think you can just map name and it'll set it as both without having to make the array for the label/value.

**Comment by ssunday** [comment]
Could there be a test for assigning automatically vs assigning a specific crafter in the applicants controller spec? Like to verify the "auto" check works?

**Comment by xray** [comment]
Changes already merged in other PR...

**Pull request closed without merging**